### PR TITLE
Feature/add size flag

### LINKED ISSUE #233: [One-off] User should be able to choose the size of the container
For instance:

```
scalingo -a appname run --size L bash
```

Things to do:

- Handle the -s/--size flag
- Send its value to the API

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #233 

Work in progress

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -64,6 +71,7 @@ var (
 			opts := apps.RunOpts{
 				App:    currentApp,
 				Cmd:    c.Args(),
+				Size:   c.String("z"),
```
Comment (Soulou) [inline cmd/run.go]: Why `z` ?

Comment (EtienneM) [inline cmd/run.go]: `s` is already for the silent flag...

Comment (Soulou) [inline cmd/run.go]: I would remove the -s flag for --silent and use it for size, it's more.. common

Comment (EtienneM) [inline cmd/run.go]: I just noticed this comment... I will modify that.

**Comment by Soulou** [comment]
I did it, its ok

**Pull request merged**